isthmus/src/main/java/io/substrait/isthmus/expression/FunctionMappings.java

```diff
@@ -34,6 +34,7 @@ public class FunctionMappings {
                 s(SqlStdOperatorTable.ASIN, "asin"),
                 s(SqlStdOperatorTable.ACOS, "acos"),
                 s(SqlStdOperatorTable.ATAN, "atan"),
+                s(SqlStdOperatorTable.ATAN2, "atan2"),
```

Comment:
Could you add this function to 
https://github.com/substrait-io/substrait-java/blob/dd069f69e9ddd7e7e668188ab535174181e61d02/isthmus/src/test/java/io/substrait/isthmus/ArithmeticFunctionTest.java#L57-L62

so we have a test for it.

Reply:
added!